### ISSUE TITLE
Version upgrade message malformed

### ISSUE THREAD

**Issue by Azami7** [zDevelopers/ImageOnMap#181]
Hello,

It looks like your version upgrade notice to Ops is malformed - it is being displayed as json. See stacktrace below and attached screenshot.

```
[21:29:36] [Server thread/ERROR]: [ImageOnMap] Exception : 
java.lang.NoSuchMethodException: net.minecraft.server.v1_16_R3.PacketPlayOutChat.<init>(net.minecraft.server.v1_16_R3.IChatBaseComponent, net.minecraft.server.v1_16_R3.ChatMessageType)
        at java.lang.Class.getConstructor0(Class.java:3349) ~[?:?]
        at java.lang.Class.getConstructor(Class.java:2151) ~[?:?]
        at fr.moribus.imageonmap.tools.text.MessageSender.sendChatPacket(MessageSender.java:588) ~[?:?]
        at fr.moribus.imageonmap.tools.text.MessageSender.sendMessage(MessageSender.java:185) ~[?:?]
        at fr.moribus.imageonmap.tools.text.MessageSender.sendSystemMessage(MessageSender.java:435) ~[?:?]
        at fr.moribus.imageonmap.tools.UpdateChecker$4.send(UpdateChecker.java:284) ~[?:?]
        at fr.moribus.imageonmap.tools.UpdateChecker$2.run(UpdateChecker.java:239) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:400) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1179) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:394) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1127) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:966) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273) ~[spigot-1.16.5.jar:3074-Spigot-9fb885e-b1064c6]
        at java.lang.Thread.run(Thread.java:829) [?:?]
```

![Screen Shot 2021-05-16 at 2 29 54 PM](https://user-images.githubusercontent.com/26971562/118413323-9b5c7280-b653-11eb-9a46-5f7648137b0b.png)

**Comment by Vlammar** [comment]
This was solved in 4.1.2.

**Comment by Azami7** [comment]
Ah looks like this was fixed after 4.0. Cool. :)
